src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -366,7 +366,7 @@ export default {
 			}
 		},
 		trackFirstRowDisplayedLoans() {
-			this.trackDisplayedLoans(this.enableRelendingExp ? 'relending' : 'recommended', 1, this.firstRowLoans);
+			this.trackDisplayedLoans('recommended', 1, this.enableRecommendedReplacementExp ? [this.featuredLoan] : this.firstRowLoans); // eslint-disable-line max-len
```

Comment:
It actually looks like we also need a fixed fallback here: https://github.com/kiva/ui/blob/048d035a642cc44bb482cdc050104b52390c05c5/src/pages/LoanFinding/LoanFinding.vue#L449

`})?.fundraisingLoans?.values[0] ?? [{ id: 0 }];` -> notice the array

Reply:
Well, with `fundraisingLoans?.values[0]` we are just returning the loan object and assigning to this `featuredLoan: {},`.
Said that, we don't need the array.
If we  want to preserve the array in both, we need to update the assignment to the `featureLoan` var.

Reply:
Oh yup, you're right.